ThePortugueseMan.Calculator/ThePortugueseMan.Calculator/Program.cs

```diff
@@ -0,0 +1,176 @@
+﻿using System.Linq;
+using CalculatorLibrary;
+
+namespace CalculatorProgram
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            bool endApp = false;
+            bool leaveList = false;
```

Comment:
Unnecessary assignments = more code. There are a few in your app, see if you can find more. I know this is the code from the tutorial, but since you have a background in coding I'll be a bit more strict 😊

![image](https://user-images.githubusercontent.com/108711476/218612936-23bf3c63-e103-482b-bb91-fd5fc27d3a5c.png)